### ISSUE TITLE
Following of this issue: #1246 option --dir dosn't work for build command 

### ISSUE THREAD

**Issue by Robin-Ln** [resend/react-email#1318]
### Describe the Bug

https://github.com/resend/react-email/issues/1246

when i execute email build --dir src/emails i get this error :

```
Error: Could not find the emails directory specified under /Users/rlouarn/git/programme-tz-2023/emails!
    at h (/Users/rlouarn/git/programme-tz-2023/.react-email/.next/server/chunks/766.js:1:4078)
Error: Could not find the emails directory specified under /Users/rlouarn/git/programme-tz-2023/emails!
    at h (/Users/rlouarn/git/programme-tz-2023/.react-email/.next/server/chunks/766.js:1:4078)
Error: Could not find the emails directory specified under /Users/rlouarn/git/programme-tz-2023/emails!
    at h (/Users/rlouarn/git/programme-tz-2023/.react-email/.next/server/chunks/766.js:1:4078)
Error: Could not find the emails directory specified under /Users/rlouarn/git/programme-tz-2023/emails!
    at h (/Users/rlouarn/git/programme-tz-2023/.react-email/.next/server/chunks/766.js:1:4078)

Error occurred prerendering page "/_not-found". Read more: https://nextjs.org/docs/messages/prerender-error
Error: Could not find the emails directory specified under /Users/rlouarn/git/programme-tz-2023/emails!
    at h (/Users/rlouarn/git/programme-tz-2023/.react-email/.next/server/chunks/766.js:1:4078)
   Generating static pages (4/6) 

Error occurred prerendering page "/". Read more: https://nextjs.org/docs/messages/prerender-error
Error: Could not find the emails directory specified under /Users/rlouarn/git/programme-tz-2023/emails!
    at h (/Users/rlouarn/git/programme-tz-2023/.react-email/.next/server/chunks/766.js:1:4078)
Error: Could not find the emails directory specified under /Users/rlouarn/git/programme-tz-2023/emails!
    at d (/Users/rlouarn/git/programme-tz-2023/.react-email/.next/server/app/preview/[slug]/page.js:1:95842)
Error: Could not find the emails directory specified under /Users/rlouarn/git/programme-tz-2023/emails!
    at h (/Users/rlouarn/git/programme-tz-2023/.react-email/.next/server/chunks/766.js:1:4078)
Error: Could not find the emails directory specified under /Users/rlouarn/git/programme-tz-2023/emails!
    at d (/Users/rlouarn/git/programme-tz-2023/.react-email/.next/server/app/preview/[slug]/page.js:1:95842)
Error: Could not find the emails directory specified under /Users/rlouarn/git/programme-tz-2023/emails!
    at h (/Users/rlouarn/git/programme-tz-2023/.react-email/.next/server/chunks/766.js:1:4078)

Error occurred prerendering page "/preview/magic-link-email.tsx". Read more: https://nextjs.org/docs/messages/prerender-error
Error: Could not find the emails directory specified under /Users/rlouarn/git/programme-tz-2023/emails!
```

### Link to the code that reproduces this issue

https://gitlab.com/robin.louarn/programme-tz-2023

### To Reproduce

nvm use
pnpm install
pnpm email:build

### Expected Behavior

generate a build correctly for deploy him on vercel

### What's your node version? (if relevant)

20.11.0

**Comment by gabrielmfern** [comment]
Thank you so much for opening the issue.

Could you also try out the latest canary? We just released a `react-email@2.1.0-canary.2` with many new fixes it could be it also closes that

**Comment by Robin-Ln** [comment]
> Thank you so much for opening the issue.
> 
> Could you also try out the latest canary? We just released a `react-email@2.1.0-canary.2` with many new fixes it could be it also closes that

yes it fix this issues :)
